src/main/java/com/akolb/HMSBenchmark.java

```diff
@@ -207,6 +208,10 @@ public static void main(String[] args) throws Exception {
               () -> benchmarkCreatePartitions(bench, client, db, tbl, instances))
           .add("dropPartitions" + '.' + instances,
               () -> benchmarkDropPartitions(bench, client, db, tbl, instances))
+          .add("renameTable",
```

Comment:
It would be good to also add microbenchmark for a single partition case both for drop and for rename.